src/python/hvor_gammel_er_du/Hvor gammel er du.md

```diff
@@ -7,30 +7,32 @@ author: Sindre O. Rasmussen, Kodeklubben Trondheim
 # Hvor gammel er du? {.intro}
 I dette oppgavesettet skal du i hvert steg lære om noen av de grunnleggende byggeklossene i python. Oppgaven gir ingen kode, men forteller hva du skal lage og hvordan du skal gjøre det. Det er derfor viktig at du leser oppgaven for å vite hva du skal gjøre. Når du har gjort alle stegene vil du sitte igjen med et program som spør hvilket år du er født og regner ut alderen din. For hver oppgave du er ferdig med bør du spørre veilederne om de kan se om du har gjort riktig. Start ved å åpne en ny blank Python fil.
 
-# Oppgave 1: Skrive ut til skjerm {.activity}
-**Hva:** Vi kan be datamaskinen om å skrive hva som helst av tekst eller tall.
+# Skrive ut til skjerm {.activity}
+- **Hva:** Be datamaskinen skrive ut tekst eller tall.
 
-**Hvordan:** Du skriver bare `print()`. Inni parentesen er det du skal skrive ut. Husk at tekst må starte og slutte med ' 
+- **Hvordan:** Funksjonen `print()` skriver ut det som er mellom `()`. Tekst må starte og slutte med `'` slik som `'denne teksten'`.
```

Comment:
Samme som endringen over, i tillegg til å legge `'` i inline kodeformatering for å hindre at den blir omformatert.